tests/bal_business_logic_suite.py

```diff
@@ -24,44 +24,112 @@
 import os
 
 from openassetio import Context, TraitsData
-from openassetio.traits.managementPolicy import ManagedTrait
 from openassetio.test.manager.harness import FixtureAugmentedTestCase
 
+
 __all__ = []
 
 
-class Test_managementPolicy_default_behavior(FixtureAugmentedTestCase):
+class LibraryOverrideTestCase(FixtureAugmentedTestCase):
     """
-    Tests that by default, the BAL manages all entities, for read, but
-    nothing for write.
+    Utility base class to load a custom BAL library for the duration of
+    the tests in a subclass.
+
+    Assumes a `_library` class attribute that gives the name of the JSON
+    file to load from the `resources` directory.
     """
 
-    __trait_sets = (
-        {"blob", "image"},
-        {"container", "shot", "frame_ranged"},
-        {"definitely_unique"},
-    )
+    _library = None
+
+    def setUp(self):
+        # TODO(TC): Extend openassetio.test.manager to allow settings to
```

Comment:
[question] What "to do"?  The first TODO is moved from below and has lingered for some time.  The second TODO I added as part of this work.  

Perhaps one or both should be a `[plan]` and an issue created and these TODOs can be removed?

The second TODO could have been done as part of this work, but I thought it was a bit out of scope(?).